recipes/pyload-ng/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyload-ng" %}
+{% set version = "0.5.0b3.dev18" %}
```

Comment:
This is a pre-release, but there is a stable release available. Could you publish the latest stable version instead?

Once you have a feedstock, you can publish prereleases in a pre-release channel. https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=pre%20release#pre-release-builds

Reply:
I think I'm better wait for a stable release on Python3. I'm not able to futurize the current stable one. https://github.com/pyload/pyload/blob/stable/module/setup.py

Reply:
OK. Are you going to add a `conda_build_config.yaml` that points this recipe to the dev channel or do you want this PR to wait until they release stable for python 3?

Reply:
@carterbox Is it possible to set the package to the dev channel without having a main one?

Reply:
I guess we will find out when this feedstock is provisioned.